src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -7,6 +7,7 @@
 using Microsoft.DotNet.DarcLib;
 using Microsoft.DotNet.Maestro.Client.Models;
 using Microsoft.Extensions.Logging;
+using Microsoft.VisualStudio.Services.Common;
```

Comment:
Is this used?

Reply:
Yeah, the GetOrAdd extension method on Dictionary.  Which is quite useful.

Reply:
I normally just add it myself per-project... weird assembly to depend on, guess it's fine if the reference is already available though.

Reply:
Yeah, we already reference the VS APis